tests/install/test.py

```diff
@@ -11,13 +13,56 @@
 from data import NETWORKS
 assert "MGMT" in NETWORKS
 
+from data import HOST_DEFAULT_PASSWORD_HASH
+from data import ISO_IMAGES
+
+def clean_files_on_pxe(mac_address):
+    pxe.server_remove_config(mac_address)
+
+def setup_pxe_boot(vm, mac_address, version):
```

Comment:
@ydirson should it be in _tests/install/conftest.py_ ?

Reply:
PXE boot can (at least in theory) be used for installed hosts too, and we may want to run other tests on such hosts one day.

Reply:
Oh you mean for an upgrade for example. But I don't see why it can stay as a function in test.py. It is not 100% clear when we should put things in conftest.py.

Reply:
not necessarily, we could have the installed host boot over the network